pandas/tests/io/excel/test_openpyxl.py

```diff
@@ -106,18 +104,18 @@ def test_write_append_mode(ext, mode, expected):
         for index, cell_value in enumerate(expected):
             assert wb2.worksheets[index]["A1"].value == cell_value
 
-
+# GH 29854
 def test_to_excel_with_openpyxl_engine(ext, tmpdir):
```

Comment:
Thanks for updating - what I meant was
```suggestion
def test_to_excel_with_openpyxl_engine(ext, tmpdir):
# GH 29854
```

Reply:
Sorry for the misunderstanding, should be fixed :)